cinema/serializers.py

```diff
@@ -1 +1,78 @@
-# write serializers here
+from rest_framework import serializers
+
+from cinema.models import Genre, Actor, CinemaHall, Movie, MovieSession
+
+
+class GenreSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Genre
+        fields = ["id", "name"]
+
+
+class ActorSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Actor
+        fields = ["id", "first_name", "last_name", "full_name"]
+
+
+class CinemaHallSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = CinemaHall
+        fields = ["id", "name", "rows", "seats_in_row", "capacity"]
+
+
+class MovieSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Movie
+        fields = ["id", "title", "description", "duration", "genres", "actors"]
+
+
+class MovieListSerializer(MovieSerializer):
+    actors = serializers.SlugRelatedField(
+        many=True,
+        read_only=True,
+        slug_field="full_name",
```

Comment:
In `MovieListSerializer`, the `slug_field` for `actors` is set to `full_name`. Since `full_name` is a property and not a database field, this might cause issues. Consider using `first_name` and `last_name` separately or ensure `full_name` is properly handled.